AIDevGallery/Styles/NavigationView.xaml

```diff
@@ -345,7 +345,7 @@
                             <SplitView.Pane>
                                 <Grid
                                     x:Name="PaneContentGrid"
-                                    HorizontalAlignment="Left"
+                                    HorizontalAlignment="Stretch"
```

Comment:
Are we still using all the resources in this resource dictionary?

Reply:
Yes